src/spikeinterface/preprocessing/common_reference.py

```diff
@@ -98,15 +98,17 @@ def __init__(
 
         # tranforms groups (ids) to groups (indices)
         if groups is not None:
-            groups = [self.ids_to_indices(g) for g in groups]
+            groups_inds = [self.ids_to_indices(g) for g in groups]
```

Comment:
```suggestion
            group_indices = [self.ids_to_indices(g) for g in groups]
```
Could you use `group_indices` instead of `gtoups_inds`?